src/endpoints/proxy/gateway.proxy.controller.ts

```diff
@@ -248,6 +248,13 @@ export class GatewayProxyController {
     }
   }
 
+  @Get('/node/waiting-epochs-left/:bls')
+  async getNodeWaitingEpochsLeft(
+    @Param('bls', ParseBlsHashPipe) bls: string,
```

Comment:
offtopic: the BLS key is not a hash, but rather a hex encoding of a public key of a validator key.

Reply:
yep, we need to do a breaking change in sdk-nestjs for that. let's put it on the task list and address this later on